app/controllers/CampaignApi.scala

```diff
@@ -23,33 +23,41 @@ class CampaignApi(components: ControllerComponents, authAction: AuthAction[AnyCo
     }
   }
 
-  def getAllCampaigns() = authAction {
-    CampaignRepository.getAllCampaigns() match {
+  def getAllCampaigns() = authAction { implicit request =>
+    val territory = Territory(request.getQueryString("territory") getOrElse "global")
+
+    CampaignRepository.getAllCampaigns(territory) match {
       case Left(JsonParsingError(error)) => InternalServerError(error)
       case Left(_)                       => InternalServerError
       case Right(campaigns)              => Ok(Json.toJson(campaigns))
     }
   }
 
-  def getBenchmarksAcrossCampaigns() = authAction {
-    CampaignService.getBenchmarksAcrossCampaigns() match {
+  def getBenchmarksAcrossCampaigns() = authAction { implicit request =>
+    val territory = Territory(request.getQueryString("territory") getOrElse "global")
```

Comment:
Could pass through as an optional param

Reply:
Global is a satisfactory default. It's Campaign Centrals default view.